test/unit/search/search_test.cpp

```diff
@@ -14,15 +14,17 @@
 
 #include "helper.hpp"
 
-using namespace seqan3;
+using seqan3::operator""_dna4;
+using seqan3::operator""_phred42;
+
 using namespace seqan3::search_cfg;
```

Comment:
Remove

Reply:
```bash
.../seqan3/test/unit/search/search_test.cpp:24:30: error: unable to find string literal operator 'operator""_dna4' with 'const char [13]', 'long unsigned int' arguments
   24 |     seqan3::dna4_vector text{"ACGTACGTACGT"_dna4};
      |                              ^~~~~~~~~~~~~~~~~~~
```
What would you do instead?

Reply:
Remove the `using namespace seqan3::search_cfg;`, not the string literals :)

(but you could do `seqan3::operator""_dna4("ACGACTGTACGTC")` to do it without `using` -- which is why we do the `using seqan3::operator""_dna4;` :))